helix-term/src/commands.rs

```diff
@@ -258,12 +258,17 @@ impl Command {
         yank, "Yank selection",
         yank_joined_to_clipboard, "Join and yank selections to clipboard",
         yank_main_selection_to_clipboard, "Yank main selection to clipboard",
+        yank_joined_to_primary_selection, "Join and yank selections to primary selection",
+        yank_main_selection_to_primary_selection, "Yank main selection to primary selection",
         replace_with_yanked, "Replace with yanked text",
         replace_selections_with_clipboard, "Replace selections by clipboard content",
+        replace_selections_with_primary_selection, "Replace selections by primary selection content",
         paste_after, "Paste after selection",
         paste_before, "Paste before selection",
         paste_clipboard_after, "Paste clipboard after selections",
         paste_clipboard_before, "Paste clipboard before selections",
+        paste_primary_selection_after, "Paste primary selection after selections",
+        paste_primary_selection_before, "Paste primary selection before selections",
```

Comment:
I think it's better to prefix them with something other than `primary_selection` since "primary selection" has a very specific meaning in helix. Maybe `primary_clipboard` ?

Reply:
Yes. Primary selection is window system term, I also thought that it can cause misunderstanding being placed into Helix code.